topics/introduction/slides/introduction.html

```diff
@@ -60,6 +66,26 @@
 
 ---
 
+## Galaxy growth
+- More than 7,000 ready to use tools for users
+- More than 7,500 [citations](https://www.zotero.org/groups/1732893/galaxy)
+- More than 350 [public Galaxy resources](https://galaxyproject.org/use/)
+-  - 120+ public servers, many more non-public
+-  - Both general-purpose and domain-specific
+
+![Galaxy timeline](../images/galaxy_timeline.png)
+
+???
+
+- The number of public galaxy resources should be regularly updated.
+
+---
+
+# Galaxy ecosystem
+
+![Galaxy ecosystem]({{ site.baseurl }}{% link shared/images/ecosystem_main_scheme.png %})
```

Comment:
OK for tweaking. I am not sure about putting this slide at the end as the purpose is to get a global view of Galaxy before digging into some part of it. Putting it at the end would sound like a resume which is not the case. Feel free to explain in more details what is your idea.

Reply:
I updated the image.

Reply:
Thanks @nagoue the updated slide is much clearer :)

My feeling here is just that this is too much information to take in for people who had never heard of Galaxy until 3 slides ago, and it feels a bit overwhelming? 

To me, this would feel more appropriate as a "Now that you know what Galaxy is, here are some advanced/related topics to explore in the other tutorials" (possibly with embedded links to related materials?) and then each instructor can use this slide at the end to go into detail about some (or none) of these topics depending on their audience. 

As an overview at the start, it might be better to simplify the graphic a bit? e.g. mention there are resources for various types of people engaging with Galaxy (researchers, devs, admins, trainers) but maybe not the second level of arrows such as API etc (this level of detail seems hard to keep complete anyway; if we mention the API as a box, we should probably also include things like workflows, visualisations, IEs, pages/reports, CVMFS etc etc and it would get very big very quickly)

what do you think?

Reply:
Another option would be to add this not to the general topic introduction slide deck, but create a slide deck associated with your new "Galaxy 101 for everyone" tutorial. I know that different people have different audiences and different ways of presenting Galaxy, and giving instructors multiple intro presentations to choose from could be a good thing for sure.

Reply:
I understand better your concern. The message from this slide is to get an global overview of what Galaxy is and show the different fields it covers.
I am not sure putting it at the end with some links to other related materials will be easy to maintain even though it would be a nice option. I may be more tempted to simplify it be removing one degree of complexity. 
The idea of adding a new introduction could be a good option too ...